addons/mrp/models/mrp_production.py

```diff
@@ -1928,6 +1928,76 @@ def action_serial_mass_produce_wizard(self):
         }
         return action
 
+    def action_split_wizard(self):
+        if len(self) > 1:
+            if any(production.state not in ('draft', 'confirmed') for production in self):
+                raise ValidationError(_("Only manufacturing orders in either the draft or confirmed status can be splitted."))
+            productions = [Command.create({'production_id': production.id}) for production in self]
+            wizard = self.env['mrp.production.split'].create({'production_ids': productions})
+            action = self.env['ir.actions.actions']._for_xml_id('mrp.action_mrp_production_split')
+            action['res_id'] = wizard.id
```

Comment:
You should not create wizard in backend. Instead add in the context {'default_production_ids': productions}